doc/manual/calling-c-and-fortran-code.rst

```diff
@@ -86,7 +83,7 @@ of an environment variable, one makes a call like this::
     julia> bytestring(path)
     "/bin/bash"
 
-Note that the argument type tuple must be written as ``(Ptr{UInt8},)``,
```

Comment:
Thanks.  I've updated the example above and following section, and will push in a second.